shared/js/project.js

```diff
@@ -316,6 +316,8 @@ var projects = module.exports = {
       case 'studio':
         if (appOptions.level.useContractEditor) {
           return 'algebra_game';
+        } else if (appOptions.skinId === 'hoc2015' || appOptions.skinId === 'infinity') {
```

Comment:
This does seem a little bit gross, as it's not really accurate to say that the current app is null when we have skinId hoc2015 or skinId infinity. Would it make more sense to call this `getStandaloneApp`? Or alternatively come up with a separate method called `getStandaloneApp` that does return null in this case?